.github/workflows/build-multiarch.yml

```diff
@@ -20,8 +20,10 @@ jobs:
         variant:
           - java20
           - java20-alpine
+          - java20-oracle-graalvm
           - java17
           - java17-graalvm-ce
```

Comment:
Please go ahead and remove the other graalvm builds.

Reply:
Okay. I'll do it except for Java 8. (Since only CE is freely distributable for Java 8.)

Reply:
Done and done.